src/main/java/core/basesyntax/strategy/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy.operation;
+
+public interface OperationHandler {
+    void operate(String fruit, int quantity);
```

Comment:
Better have Transaction as an argument. We should have as few as possible arguments.
```suggestion
    void operate(FruitTransaction transaction);
```